### PR TITLE
Unitary Tests for components BoxOverlay, Button, Close button & Dot

### PR DESCRIPTION
**Pull request by gemanepa**

### CODE REVIEW

File: src/components/BoxOverlay/BoxOverlay.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import cn from 'classnames';
 import { useKey } from 'react-use';
 import { useHistory } from 'react-router-dom';
 
-import { CloseButton } from 'components/CloseButton';
+import { CloseButton } from '../CloseButton';
```
Comment (RusinovAnton) [inline src/components/BoxOverlay/BoxOverlay.js]: I would rather add support of absolute imports from `src/` by putting the following rule into `package.json`:

```json
"jest": {
    "moduleDirectories": ["node_modules", "src"],
```

Refs: https://github.com/kulshekhar/ts-jest/issues/64#issuecomment-264272241

**Comment by RusinovAnton** [comment]
Great job overall, thank you so much for this contribution!

File: src/components/BoxOverlay/BoxOverlay.js (hunk 1 above)
Comment (gemanepa) [reply src/components/BoxOverlay/BoxOverlay.js]: Done

File: src/data/normalizeDocsIndex.js [hunk 2]
```diff
@@ -21,7 +21,10 @@ module.exports = index => {
     const {
       name: domainName,
       path: domainPath,
-      properties: { prototype: prototypeProperties, static: staticProperties } = {},
```
Comment (gemanepa) [inline src/data/normalizeDocsIndex.js]: @RusinovAnton I tried with no use to let this file as it was before the merge conflict but weirdly all the commits I made to put the four lines in one were pushed as empty
Screenshot from 2019-12-07 00-42-30

**Comment by gemanepa** [comment]
@RusinovAnton done with the changes you suggested. I solved the merge conflict but I commented something in src/data/normalizeDocsIndex.js, look at it please

File: src/data/normalizeDocsIndex.js (hunk 2 above)
Comment (RusinovAnton) [reply src/data/normalizeDocsIndex.js]: Do you mean the formatting? I've recently added the "pretty-quick" library which applies prettier formatting automatically on the pre-commit hook, so to me, it looks like that is what's happening which is totally ok

**Pull request merged**